mysql-test/suite/innodb/t/percona_extended_innodb_status.test

```diff
@@ -66,3 +66,56 @@ DROP TABLE t;
 --echo Grepping InnoDB status for $SEARCH_PATTERN
 --source include/search_pattern_in_file.inc
 --remove_file $innodb_status
+
+
+--echo #
+--echo # PS-6150: MySQL crash - sync_mutex_to_string
+--echo #
+
+connect (con1,localhost,root,,);
+connect (con2,localhost,root,,);
```

Comment:
count_session.inc and wait_until_count_session.inc

Reply:
done